docs/getting-started-guides/logging-elasticsearch.md

```diff
@@ -0,0 +1,222 @@
+kube-dns                k8s-app=kube-dns,kubernetes.io/cluster-service=true,kubernetes.io/name=KubeDNS                      k8s-app=kube-dns                10.0.0.10      53/UDP
+                                                                                                                                                                           53/TCP
+kubernetes              component=apiserver,provider=kubernetes                                                             <none>                          10.0.0.1       443/TCP
+monitoring-grafana      kubernetes.io/cluster-service=true,kubernetes.io/name=Grafana                                       k8s-app=influxGrafana           10.0.167.139   80/TCP
+monitoring-heapster     kubernetes.io/cluster-service=true,kubernetes.io/name=Heapster                                      k8s-app=heapster                10.0.208.221   80/TCP
+monitoring-influxdb     kubernetes.io/cluster-service=true,kubernetes.io/name=InfluxDB                                      k8s-app=influxGrafana           10.0.188.57    8083/TCP
+```
+
+By default two Elasticsearch replicas are created and one Kibana replica is created.
+
+```
+$ kubectl get rc
+CONTROLLER                     CONTAINER(S)            IMAGE(S)                                          SELECTOR                                   REPLICAS
+elasticsearch-logging-v1       elasticsearch-logging   gcr.io/google_containers/elasticsearch:1.4        k8s-app=elasticsearch-logging,version=v1   2
+kibana-logging-v1              kibana-logging          gcr.io/google_containers/kibana:1.3               k8s-app=kibana-logging,version=v1          1
+kube-dns-v3                    etcd                    gcr.io/google_containers/etcd:2.0.9               k8s-app=kube-dns,version=v3                1
+                               kube2sky                gcr.io/google_containers/kube2sky:1.9                                                        
+                               skydns                  gcr.io/google_containers/skydns:2015-03-11-001                                               
+monitoring-heapster-v4         heapster                gcr.io/google_containers/heapster:v0.14.3         k8s-app=heapster,version=v4                1
+monitoring-influx-grafana-v1   influxdb                gcr.io/google_containers/heapster_influxdb:v0.3   k8s-app=influxGrafana,version=v1           1
+                               grafana                 gcr.io/google_containers/heapster_grafana:v0.7                                         
+```
+
+The Elasticsearch and Kibana services are not directly exposed via a publically reachable IP address. Instead,
+they can be accessed via the service proxy running at the master. The URLs for accessing Elasticsearch
+and Kibana via the service proxy can be found using the `kubectl cluster-info` command.
+
+```
+$ kubectl cluster-info
+Kubernetes master is running at https://146.148.94.154
+Elasticsearch is running at 
```

Comment:
Fixed (after checking that this was an editor fumble and that cluster-info does indeed do the right thing).